### PR TITLE
base docker image changed to cloudformation-core

### PR DESCRIPTION
**Pull request by iiman**
## Approvals

- [x] Verified
- [x] @TimMoore
- [x] @hebbo
- [ ] @ptolemy428
- [x] @iiman

Uses cloudformation-core docker image as base. 

See [this card](https://trello.com/c/ZYAKMM9o/447-update-docker-ansible-to-use-cloudformation-core-docker-image) for further discussion

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM python:2.7.11-wheezy
-MAINTAINER SRE Team <techops@cultureamp.com>
+FROM cultureamp/cloudformation-core:iy-mv_utils_from_cfnstacks
```
Comment (iiman) [inline Dockerfile]: This pulls `iy-mv_utils_from_cfnstacks` branch. Once [this PR](https://github.com/cultureamp/cloudformation-core/pull/2) is merged to master, we will pull from `latest`

**Comment by iiman** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;ZYAKMM9o&#x2F;447-update-docker-ansible-to-use-cloudformation-core-docker-image"> Update docker-ansible to use cloudformation-core docker image</a>

**Comment by hebbo** [comment]
👍

File: Dockerfile (hunk 1 above)
Comment (TimMoore) [inline Dockerfile]: Should be safe to pull from latest now.

**Comment by TimMoore** [comment]
:shipit:

**Pull request merged**